### PR TITLE
Release/0.18.0

### PR DESCRIPTION
**Pull request by cassava**
I think we are just about ready to "release" 0.18.0.

Note: we are not actually releasing any binaries, this is just for bumping the version and tagging it.

### CODE REVIEW

File: docs/develop/building-cloe.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ directory of every Cloe package (e.g. in ``plugins/basic/``):
     output of ``make help`` in the respective package directory for a list of
     the available targets (here: ``make -C ./models help``):
 
-    .. runcmd:: make -C ../models help
+    .. runcmd:: make -C ../../models help
```
[the remark this answers is not in the document]
Comment (cassava) [reply docs/develop/building-cloe.rst:29]: That's interesting, but makes sense. Other commands like `literalinclude` have CWD in the directory of the file, but `runcmd` seems to be from the `docs` directory.

File: docs/reference/plugins/minimator.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Minimator Simulator
+===================
+
+.. todo:: Describe the minimator simulator.
+
+Defaults
+--------
+
+The following help can be viewed with :command:`cloe-engine usage basic`:
```
Comment (tobifalk) [inline docs/reference/plugins/minimator.rst:11]: should be `minimator` instead of `basic`

File: docs/reference/plugins/minimator.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+.. literalinclude:: basic.yaml
+   :language: yaml
+
+JSON Schema
+-----------
+
+The following can be viewed with :command:`cloe-engine usage --json basic`:
```
Comment (tobifalk) [inline docs/reference/plugins/minimator.rst:19]: should be `minimator` instead of `basic`

File: docs/reference/plugins/minimator.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+.. literalinclude:: basic_schema.json
```
Comment (tobifalk) [inline docs/reference/plugins/minimator.rst:21]: should be `minimator` instead of `basic`

**Review by tobifalk — CHANGES_REQUESTED** [review]
Apart from the few minor findings it looks good!

File: docs/reference/plugins/minimator.rst (hunk 4 above)
Comment (cassava) [reply docs/reference/plugins/minimator.rst:21]: Oops.

**Review by tobifalk — APPROVED** [review]
Great work!

**Pull request merged**